rest_framework/fields.py

```diff
@@ -1351,9 +1351,11 @@ class DurationField(Field):
         'overflow': _('The number of days must be between {min_days} and {max_days}.'),
     }
 
-    def __init__(self, **kwargs):
+    def __init__(self, format=empty, **kwargs):
```

Comment:
We should disallow specifying this as positional argument: `DurationField('iso-8601')` (we don't want that).

All other options come from the `**kwargs`...